tests/customer-resource-show-test.js

```diff
@@ -166,19 +172,19 @@ describeApplication('CustomerResourceShow', () => {
       });
 
       return this.visit(`/eholdings/customer-resources/${resource.id}`, () => {
-        expect(ResourcePage.$root).to.exist;
+        expect(ResourcePage).to.exist;
       });
     });
 
     it('displays the title name', () => {
-      expect(ResourcePage.titleName).to.equal(resource.title.name);
+      expect(ResourcePage.titleName).to.equal('Best Title Ever');
     });
 
-    it.still('does not display a content type', () => {
-      expect(ResourcePage.contentType).to.equal('');
+    it('does not display a content type', () => {
+      expect(ResourcePage.hasContentType).to.be.false;
     });
 
-    it.still('does not display a publication type', () => {
+    it('does not display a publication type', () => {
```

Comment:
Same thing here with `it.always` although maybe it doesn't apply in this case because it ensures that there _is_ a publication type element in the DOM, but it is empty. My concern would be if the publication type element renders at least once before any actual text content is rendered inside. In that case it would pass before the text content gets rendered.

Reply:
Changed the logic in the view to conditionally show the Publication type div, then switched this to an `it.always` that checks for `hasPublicationType` (using `isPresent`).